src/view/bitcoin-units.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import { observer } from 'mobx-react';
+import PropTypes from 'prop-types';
+import Background from '../component/background';
+import MainContent from '../component/main-content';
+import { Header, Title } from '../component/header';
+import { Button, BackButton } from '../component/button';
+import { ListItem } from '../component/list';
+import Text from '../component/text';
+import { color } from '../component/style';
+
+//
+// Bitcoin Units View
+//
+
+const styles = StyleSheet.create({
+  content: {
+    paddingTop: 35,
+    paddingLeft: 50,
+    paddingRight: 50,
+  },
+});
+
+const BitcoinUnitsView = ({ store, nav, setting }) => {
+  return (
+    <Background color={color.blackDark} style={styles.wrapper}>
+      <Header separator>
+        <BackButton onPress={() => nav.goHome()} />
+        <Title title="Bitcoin Units" />
+        <Button disabled onPress={() => {}} />
+      </Header>
+      <MainContent style={styles.content}>
+        <UnitsListItem
+          name="Satoshi   (0.00000001 BTC)"
+          type="sat"
+          selectedUnit={store.settings.unit}
+          onSelect={() => setting.setBitcoinUnit('sat')}
+        />
+        <UnitsListItem
+          name="Bits   (0.000001 BTC)"
+          type="bit"
+          selectedUnit={store.settings.unit}
+          onSelect={() => setting.setBitcoinUnit('bit')}
+        />
+        <UnitsListItem
+          name="Bitcoin"
+          type="btc"
+          selectedUnit={store.settings.unit}
+          onSelect={() => setting.setBitcoinUnit('btc')}
+        />
```

Comment:
Probably best to wrap the three UnitsListItems in a View and center that like in the design.

![image](https://user-images.githubusercontent.com/1374174/41839810-79403b6c-7864-11e8-8c42-4529e16303be.png)